app/src/serviceWorker.js

```diff
@@ -1,134 +0,0 @@
-// This optional code is used to register a service worker.
```

Comment:
Do we not need a service worker?

Reply:
No this is just something that came out of the box with create react app and was interfering with some of the functionality locally. CRA doesn't even include service worker code anymore.